.golangci.yml

```diff
@@ -6,4 +6,5 @@ linters:
 
 linters-settings:
   lll:
-    line-length: 200
+    line-length: 100
+    tab-width: 1
```

Comment:
how about adding `statements:75` too

Reply:
I could not find this settings in https://github.com/golangci/golangci-lint/blob/2ea496f22b997490dd43bbb1536f28908b4389c6/pkg/config/linters_settings.go#L337

Reply:
I think it is from number of lines for FunLength: https://github.com/golangci/golangci-lint/blob/2ea496f22b997490dd43bbb1536f28908b4389c6/pkg/config/linters_settings.go#L200